### ISSUE TITLE
add tweetynet + fix crowsetta/vak links?

### ISSUE THREAD

**Issue by NickleDave** [rhine3/bioacoustics-software#1]
Hi @rhine3 thank you again for putting together this great resource

* Wanted to ask if you'd accept a PR adding tweetynet?
https://github.com/yardencsGitHub/tweetynet

The model both segments and classifies the segments -- diff't people have diff't schema but I think on the README it would make sense under classification

* Also wanted to ask about changing the links to `crowsetta` and `vak` -- we set up a GitHub org so they live here now: https://github.com/vocalpy
Again I'm happy to make a quick pull request to change them just to save you the effort

Last thing: very minor but I don't think AVA does detection or segmenting per se? (You have it under "detection"). I say this because they used `tweetynet` with `vak` to do some of the segmenting (see [methods of eLife paper](https://elifesciences.org/articles/67855#s4)).  The main goal of AVA as I understand it is to do dimensionality reduction by mapping existing sounds/segments to a latent space with fewer dimensions. Similar (but not the same) as models from Tim Sainburg (https://github.com/timsainb/avgn_paper). Maybe you could consider adding a section like "Unsupervised / clustering / dimensionality reduction"?

**Comment by rhine3** [comment]
Hi @NickleDave of course I'd accept a PR making these changes! Thanks for your contributions in software development and in keeping this list up to date.

Also, I'm transitioning this list to store the information in a table, because the current document format of the list with software in multiple places is very hard to keep updated. Perhaps one day I/other folks can use that table to create various outputs like a more easily filterable website or an automatically updated README. Currently the table is in a linked Google Sheet. If you could make these updates on the table, I would appreciate it quite a bit, as the table currently represents the most updated information. I will give you access to edit the spreadsheet if you email me your preferred Google account.

If you would be willing to add a section including unsupervised/clustering/dimensionality reduction software, I would appreciate it. Currently that software is lumped into other categories. I don't plan on making a big overhaul to this list soon, so if you don't add that category, it will be something I keep in mind for the future.

**Comment by NickleDave** [comment]
> of course I'd accept a PR making these changes! 

Great, will do

> I will give you access to edit the spreadsheet if you email me your preferred Google account.

Sure, also will do

> If you would be willing to add a section including unsupervised/clustering/dimensionality reduction software, I would appreciate it. 

Also also will do :slightly_smiling_face:

**Comment by NickleDave** [comment]
Do you want to make another issue re: creating various outputs like a more easily filterable website or an automatically updated README?

I could contribute some ideas there. I think that there's a lot of cases where researchers would like an out-of-the box solution to generate a website from a schema without needing to do web dev stuff like databases and deploying apps.
I wonder if something like what @vsoch has done with RSEPedia could be a solution
https://github.com/rseng/rseng
Basically the "database" lives in this repo, made out of .json files. You could do something similar with your spreadsheet
https://github.com/rseng/software

**Comment by vsoch** [comment]
Oh indeed, and I'd love to help with that! It's actually fairly easy to create automation with a static (filesystem based) "database" and even a static API! And actually if you already have it in a spreadsheet we could use that for automation - I did that a few times for USRSE stuffs.

Let me know your vision / what you'd like and I'd be happy to take a first shot! :t-rex:

**Comment by NickleDave** [comment]
Thanks @vsoch don't you have some ABIs to analyze? :smirk_cat: very kind of you to offer :heart:

**Comment by vsoch** [comment]
haha, you know me, I always am overly caught up with "that work stuff" and looking for fun projects!!

**Comment by vsoch** [comment]
Anyhoo - let me know what you'd like @rhine3 - whether it's abstract, a "point to another" site that you like, or just some simple automation. It will be fun to do I think.

**Comment by rhine3** [comment]
Oops, I guess I didn't need to reopen this to say I made a new issue for this - #3! It is so generous of you @vsoch to offer to help build it--that would be incredible. There is a [spreadsheet](https://docs.google.com/spreadsheets/d/1Ba1MY4o5Sm1f08IekJcbxAtSjkDN71Z1RZ42kzrofJ0/edit#gid=0) we can use. Maybe we can email or set up a call to discuss? I'll be away much of next week but would be available the following week.

**Comment by vsoch** [comment]
That sounds great! I'll send you an email to follow up.
